app/assets/stylesheets/admin/customers.css.scss

```diff
@@ -1,3 +1,14 @@
 .tag-with-rules {
   color: black;
 }
+
+table#customers.index {
+
+  tr.customer {
+
+    .guest-label {
+      color: #999;
+      display: block;
```

Comment:
I'd also use a smaller font size as it is extra information at a different hierarchy level as the label.

Reply:
Done.